veterans/components/common/header/banner/Banner.tsx

```
@@ -0,0 +1,33 @@
+  height: 34;
+};
+
+type BannerProps = RazomProps;
+
+function Banner({ name, height }: BannerProps) {
+  return (
+    <div className={st.bannerContainer}>
+      <Link
+        href="/"
+        className={st.bannerLink}
+        {...(false ? { prefetch: false } : {})}
```

Biome (1.9.4)

[error] Unexpected constant condition.



(lint/correctness/noConstantCondition)